src/components/atoms/Button/Button.tsx

```diff
@@ -102,10 +102,10 @@ const SButton = styled.button<IButtonProps>`
   align-items: center;
   width: ${props => props.fullWidth && '100%'};
   padding: ${({ sizing }) => sizing && paddings[sizing]};
-  font-family: ${openSans};
+  font-family: ${typography.primary};
   font-size: ${({ sizing }) => sizing && fontSizes[sizing]};
   line-height: ${({ sizing }) => (sizing === 'large' ? '30px' : '24px')};
-  font-weight: 500;
+  font-family: ${typography.weights.semibold};
```

Comment:
@lluia should this be font-weight?

Reply:
Ahhh you catch 💯 , yez should be `font-weight` 👍